app/Controllers/importExportController.php

```diff
@@ -256,11 +262,11 @@ private function ttrssXmlToJson(string $xml) {
 	/**
 	 * This method import a JSON-based file (Google Reader format).
 	 *
-	 * @param string $article_file the JSON file content.
-	 * @param boolean $starred true if articles from the file must be starred.
+	 * $article_file the JSON file content.
+	 * true if articles from the file must be starred.
 	 * @return boolean false if an error occurred, true otherwise.
 	 */
-	private function importJson($article_file, $starred = false) {
+	private function importJson(string $article_file, ?bool $starred = false): bool {
```

Comment:
```suggestion
	private function importJson(string $article_file, bool $starred = false): bool {
```

I would need to check in more details, but I believe optional parameters are not supposed to have a `?` nullable type. When a `null` is passed, the default boolean value is assigned. So in our function, we always expect a boolean, not a nullable boolean.

Reply:
done